far/headers.hpp

```diff
@@ -82,6 +82,7 @@ WARNING_POP()
 #include <forward_list>
 #include <fstream>
 #include <functional>
+#include <generator>
```

Comment:
Som compilers seem to be unhappy with `generator`. Can we upgrade them? If not, I'll fix it.

Reply:
OK. Let's do without.